Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -385,8 +381,15 @@ void setHeadingSetpoint(int setpoint){
 void inputCapture(){
     int* channelIn;
     channelIn = getPWMArray(getTime());
-    inputMixing(channelIn, &input_RC_RollRate, &input_RC_PitchRate, &input_RC_Throttle, &input_RC_YawRate, &input_RC_Flap);
-
+    
+    inputMixing(channelIn, &input_RC_RollRate, &input_RC_PitchRate, &input_RC_Throttle, &input_RC_YawRate);
+    
+#if VEHICLE_TYPE == FIXED_WING
```

Comment:
I dont think AttitudeManager should be aware of the vehicle type we're using, so we shouldnt have these defines. In this case, it'd probably be better if inputMixing takes in either a struct, and each vehicle file can choose to either populate or not populate the Flap field.